### PR TITLE
Fixed nofitications and pushbullet!

### PR DESCRIPTION
**Pull request by malosaa**
This pull request includes a
- [x] Bug fix
- [x] New feature
- [ ] Translation

The following changes were made

map details are better , and fixed errors
## 
## 
## 

If this is related to an existing ticket, include a link to it as well.

### CODE REVIEW

File: notifier.py [hunk 1]
```diff
@@ -22,8 +19,9 @@ def init():
         wanted_pokemon = [a.lower() for a in wanted_pokemon]
         # get api key
         api_key = _str( data["pushbullet"] )
-        if api_key:
-            pushbullet_client = Pushbullet(api_key)
+        pushbullet_client = Pushbullet(api_key)
+        if data.get("channel_tag"):
+            pushbullet_channel=get_channel(pushbullet_client, _str( data.get("channel_tag") ))
```
Comment (jxmorris12) [inline notifier.py:24]: change this to just `str()` without the underscore. sorry I know it's confusing

**Comment by malosaa** [comment]
fixed today

**Comment by Silvawar14** [comment]
This gmaps hyperlink (as well as the current one) works for me on my desktop but not on mobile.

The old hyperlink of maps.google.com/maps?q=(lat,lon,zoom) worked great.

**Comment by malosaa** [comment]
Here on my phone and iPad iT works ,

Op zaterdag 23 juli 2016 heeft Silvawar14 notifications@github.com het
volgende geschreven:

**Comment by malosaa** [comment]
Check Here

https://github.com/jxmorris12/PokemonGo-Finder/pull/32

Op zaterdag 23 juli 2016 heeft Silvawar14 notifications@github.com het
volgende geschreven:

**Pull request closed without merging**